src/container/Container.js

```diff
@@ -127,11 +125,13 @@
 	 */
 	p._internalOpen = function(path, options)
 	{
-		options = $.extend(
-		{
-			singlePlay: false,
-			playOptions: null
-		}, options);
+		options = Object.assign(
```

Comment:
Alas, `Object.assign` is not supported in IE 11... I believe SpringRoll has a core polyfill called `Object.merge` that could resolve this. Or, you could just hand-write it:
```javascript
options = options || {};
options.singlePlay = options.singlePlay !== undefined ? options.singlePlay : false;
options.playOptions = options.playOptions !== undefined ? options.playOptions : null
```